apps/editor/src/app/base.tsx

```diff
@@ -478,8 +478,7 @@ export function Base({children}: BaseProps) {
                     </PermissionControl>
                   </Nav.Menu>
                 </PermissionControl>
-                <PermissionControl
-                  qualifyingPermissions={['CAN_GET_SETTINGS', 'CAN_UPDATE_SETTINGS']}>
+                <PermissionControl qualifyingPermissions={['CAN_UPDATE_SETTINGS']}>
```

Comment:
Is that on purpose that the CAN_GET_SETTINGS isn't needed anymore?

Reply:
Then we should remove CanGetSettings from the editor's permission instead. There's other places (like the actual component) where that permission is also used.